fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java

```diff
@@ -293,7 +294,15 @@ public static void setSystemVariable(SessionVariable sessionVariable, SystemVari
         if (SessionVariable.DEPRECATED_VARIABLES.stream().anyMatch(c -> c.equalsIgnoreCase(setVar.getVariable()))) {
             return;
         }
-        checkSystemVariableExist(setVar);
+        try {
+            checkSystemVariableExist(setVar);
+        } catch (DdlException e) {
+            if (Config.ignore_invalid_system_variable) {
+                return;
```

Comment:
Would suggest adding a WARN log here.

Reply:
@pedrong This is a temp draft, move to this pr https://github.com/StarRocks/starrocks/pull/37177